pkgs/development/python-modules/datrie/default.nix

```diff
@@ -17,17 +19,24 @@ buildPythonPackage rec {
     hash = "sha256-UlsI9jjVz2EV32zNgY5aASmM0jCy2skcj/LmSZ0Ydl0=";
   };
 
+  patches = [
+    # https://github.com/pytries/datrie/pull/89
+    (fetchpatch {
+      name = "remove-pytest-runner.patch";
+      url = "https://github.com/pytries/datrie/commit/c67d07cb7cf6c05e20e9e750452ffd35b1a2cfbf.patch";
+      hash = "sha256-QEj4+PihO//e7daK5FIf81sItQihfOR6jzI1VSVrFTM=";
+    })
+  ];
+
   nativeBuildInputs = [
     setuptools
+    wheel
```

Comment:
I don't think we should add a not used yet dependency. Also, adding it when necessary will be a trivial task.

Reply:
Thank you for reviewing! That sounds good, I can sit on this change until I'm closer to pushing everything. For reference, https://github.com/NixOS/nixpkgs/pull/245509 is the overarching work.

Reply:
This is already required and just through some impurities ignored, see https://github.com/pytries/datrie/blob/master/pyproject.toml#L4